test/jit/test_upgraders.py

```diff
@@ -37,6 +37,23 @@ def f():
         self.assertTrue(upgraders_size == upgraders_size_second_time)
         self.assertTrue(upgraders_dump == upgraders_dump_second_time)
 
+    def test_add_value_to_version_map(self):
```

Comment:
This method seems to be testing the test-only methods themselves. Could we use the `_add_test_entry` method to actually test other part of upgrader processing code? Like registering an upgrader and apply it on a piece of old IR, which gets upgraded etc.

Reply:
Ping

Reply:
These PRs come before the PRs where we actually replace the old IR. (https://github.com/pytorch/pytorch/pull/68358)
In there, I will be using these utility methods to test our change.